samtools.wdl

```diff
@@ -452,6 +452,46 @@ task Merge {
     }
 }
 
+task Quickcheck {
+    input {
+        File inputBam
+
+        Int threads = 1
+        Int memoryGb = 1
```

Comment:
Shall we be bold and also hardcode this in the runtime section? Given that BGZip chunks only contain 65K of data and only the header and footer of the file are checked 1GB should be ample.